src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+    </div>
+    <div class="stars stars--1">
```

Comment:
add an empty line between _multiline_ siblings (code styling). Apply for other cases as well
```suggestion
    </div>
    
    <div class="stars stars--1">
```